packages/components/virtual-list/src/hooks/use-grid-wheel.ts

```diff
@@ -24,7 +24,16 @@ export const useGridWheel = (
       (x <= 0 && atXStartEdge.value) || (x >= 0 && atXEndEdge.value)
     const yEdgeReached =
       (y <= 0 && atYStartEdge.value) || (y >= 0 && atYEndEdge.value)
-    return xEdgeReached && yEdgeReached
+    if(x !== 0 && y === 0 && !xEdgeReached){
```

Comment:
hi, I think `y === 0` is not necessary here. I guess your idea is that if `x` is changed and not reached that should return `false` to prevent default event. If `x` is changed and reached that should return `true` to permit default scroll event.

And if both `x` and `y` are changed but only one reaches, I think the default event should also be blocked. So, the same as the following `x === 0` I think is not necessary either.